pages/data-providers/finalisation.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Message } from '@oacore/design'
+
+import RegistrationLayout from '../../modules/registration-layout'
+
+import { withGlobalStore } from 'store'
+
+const Finalisation = ({ store: { dataProvider } }) => (
+  <RegistrationLayout>
+    <Message variant="success">
+      Data provider {dataProvider.created?.oaiPmhUrl} has been successfully
+      added to our DB. The harvesting process will start soon.
+    </Message>
```

Comment:
There should be 'add one more' button that resets the form. Can be added later, keep it in mind.

I am not sure about route transitioning but this can be considered later too.